src/google/google-calendars/googlecalendarsyncadaptor.cpp

```diff
@@ -1259,9 +1356,9 @@ void GoogleCalendarSyncAdaptor::upsyncFinishedHandler()
                 // update this event in the local calendar
                 m_storage->loadNotebookIncidences(googleNotebook->uid());
                 m_storage->load(kcalEventId);
```

Comment:
Hrm, actually this may cause issues... I'll test it again, but ISTR one case where you had to explicitly load an incidence by UID in order to be able to delete it from the calendar (and presumably update it also).